packages/plugins/integration-make-com/src/lib/make-com.service.ts

```
@@ -1,142 +1,466 @@
 
 	/**
-	 * Upsert a Make.com integration setting for a tenant.
+	 * Makes an authenticated API call to Make.com.
 	 *
-	 * If the setting exists, its value is updated. If it does not exist, a new setting is created.
+	 * @param {string} makeApiUrl - The Make.com API endpoint.
+	 * @param {string} method - The HTTP method to use.
+	 * @param {any} data - The data to send with the request.
+	 * @returns {Promise<any>} - The API response.
+	 */
+	async makeApiCall(makeApiUrl: string, method: string = 'GET', data: any = null, retryLimit = 1): Promise<any> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)